Doc/library/functions.rst

```diff
@@ -711,8 +711,11 @@ are always available.  They are listed here in alphabetical order.
 
    Return an integer object constructed from a number or string *x*, or return
    ``0`` if no arguments are given.  If *x* is a number, return
-   :meth:`x.__int__() <object.__int__>`.  For floating point numbers, this
-   truncates towards zero.
+   :meth:`x.__int__() <object.__int__>`. If *x* defines
```

Comment:
Since you're updating this paragraph anyway, it makes sense to mention `__index__` as well, perhaps by converting to a bulleted list of conversion methods in order of priority: `x.__index__()`, `x.__int__()`, `x.__trunc__()`.